wpilib/wpilib/drive/differentialdrive.py

```diff
@@ -0,0 +1,210 @@
+# validated: 2017-10-23 TW 19addb04cf4a edu/wpi/first/wpilibj/drive/DifferentialDrive.java
+# ----------------------------------------------------------------------------
+# Copyright (c) FIRST 2008-2017. All Rights Reserved.
+# Open Source Software - may be modified and shared by FRC teams. The code
+# must be accompanied by the FIRST BSD license file in the root directory of
+# the project.
+# ----------------------------------------------------------------------------
+
+import hal
+import math
+from .robotdrivebase import RobotDriveBase
+
+__all__ = ["DifferentialDrive"]
+
+
+class DifferentialDrive(RobotDriveBase):
+    """A class for driving differential drive/skid-steer drive platforms such as the Kit of Parts drive
+        base, "tank drive", or West Coast Drive.
+
+        These drive bases typically have drop-center / skid-steer with two or more wheels per side
+        (e.g., 6WD or 8WD).
+
+        A differential drive robot has left and right wheels separated by an arbitrary width.
+
+        Drive base diagram:
+
+        |_______|
+        | |   | |
+          |   |
+        |_|___|_|
+        |       |
```

Comment:
Something gives me the feeling this was meant to be rendered preformatted. It isn't.

Reply:
?